pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/filter/query/MultipleMatchQueryStrategy.java

```diff
@@ -5,14 +5,38 @@
 import org.elasticsearch.index.query.QueryBuilder;
 
 public class MultipleMatchQueryStrategy extends BaseQueryStrategy {
+    public static String FIELD_NAME_ANSWER = "activities.questionsAnswers.answer";
+    public static String FIELD_NAME_GROUPED_OPTIONS = "activities.questionsAnswers.groupedOptions.";
 
     @Override
     protected QueryBuilder getMainQueryBuilderFromChild(BaseQueryBuilder baseQueryBuilder) {
         BoolQueryBuilder boolQueryBuilder = new BoolQueryBuilder();
         Object[] values = baseQueryBuilder.payload.getValues();
         for (Object value : values) {
-            boolQueryBuilder.should(new MatchQueryBuilder(baseQueryBuilder.payload.getFieldName(), value));
+            String[] fieldNameAndValue =getCorrectFieldNameAndValues(baseQueryBuilder.payload.getFieldName(), (String) value);
+            boolQueryBuilder.should(new MatchQueryBuilder(fieldNameAndValue[0], fieldNameAndValue[1]));
         }
         return boolQueryBuilder;
     }
+
+    private String[] getCorrectFieldNameAndValues(String fieldName, String value){
+        String[] fieldNameAndValue = new String[2];
+        if (!FIELD_NAME_ANSWER.equals(fieldName) || !value.contains(".")){
```

Comment:
if it is not `questionAnswer.answer` we should not touch it
also if value doesn't have a `.` in it it should not get changed